Makefile

```diff
@@ -1,22 +1,11 @@
-include build.mk
+include scripts/build.mk
```

Comment:
Big fan of this but just not that you might hit some merge conflicts soon. Shouldn't be an issue but just FYI